lib/src/components/_stacks-tables.less

```diff
@@ -262,6 +267,13 @@
         color: @fc-light;
         cursor: pointer;
 
+        //  If an <a> link is used, it should appear like a <th> and not an <a>
+        th {
+            a {
```

Comment:
Since there's only one descendent here, I don't think we need the nesting. This would probably do it? 2/10 on this one, heh

``` css
th a {
  color: inherit;
}
```

Reply:
👍 I'm a 1/10 so I will change.